medmodels/statistic_evaluations/comparer/data_comparer.pyi

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
+
+from typing import Dict, List, Tuple, TypedDict, Union
+
+from medmodels.medrecord.medrecord import MedRecord
+from medmodels.medrecord.querying import NodeOperation
+from medmodels.medrecord.types import (
+    AttributeInfo,
+    AttributeSummary,
+    Group,
+    GroupInputList,
+    MedRecordAttribute,
+    MedRecordAttributeInputList,
+    NodeIndex,
+)
+
+class Cohort:
```

Comment:
Should probably be a `TypedDict` if there is no other functionality implemented here